scripts/fast-reboot

```diff
@@ -327,6 +333,23 @@ function check_docker_exec()
     done
 }
 
+function check_db_integrity()
+{
+    if [[ "$REBOOT_TYPE" = "warm-reboot" || "$REBOOT_TYPE" = "fastfast-reboot" ]]; then
+        CHECK_DB_INTEGRITY=0
+        /usr/local/bin/check_db_integrity.py || CHECK_DB_INTEGRITY=$?
+        if [[ CHECK_DB_INTEGRITY -ne 0 ]]; then
+            if [[ x"${IGNORE_DB_CHECK}" == x"yes" ]]; then
```

Comment:
is there a need to even execute this script if ignore_db is set to True?

Reply:
This is in accordance with the other checks in this script (OA RESTARTCHECK test, ASIC checksum match test) which run the check, and then decide to ignore or fail.
I think this approach is still beneficial - we log the failure for future debugging and then ignore it. At any point after reboot, one can go and see what happened in the going down path.